src/components/SelectedAnnotation.jsx

```diff
@@ -192,12 +192,12 @@ class SelectedAnnotation extends React.Component {
   closePrompt() {
     this.props.dispatch(togglePopup(
       <QuestionPrompt
-        confirm="Yes, close without saving"
-        deny="No, continue transcribing"
+        confirm={this.props.translate('closeAnnotation.confirm')}
```

Comment:
Maybe `const { translate } = this.props;` at the top of this function, to save some repetition in these props.

Reply:
Ah, good call. This will help readability.